packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/RecentSubjects.js

```diff
@@ -15,6 +13,9 @@ counterpart.registerTranslations('en', en)
 function RecentSubjects (props) {
   const { isLoggedIn, recents, projectName, size, slug } = props
   const height = (size === 1) ? '40vw' : '200px'
+  const placeholderUrl = process.env.ASSET_PREFIX
+    ? `${process.env.ASSET_PREFIX}/subject-placeholder.png`
```

Comment:
Isn't this still  https://frontend.preview.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/classify which resolves to https://static.zooniverse.org/preview.zooniverse.org/panoptes-front-end/master/static/subject-placeholder.png.

Reply:
nope. the kube config for staging sets `ASSET_PREFIX` to `https://fe-project.preview.zooniverse.org`, so this becomes `https://fe-project.preview.zooniverse.org/subject-placeholder.png`, which gets resolved.

Prepending the `ASSET_PREFIX` without using the new `public` folder would work too: https://fe-project.preview.zooniverse.org/static/subject-placeholder.png

Reply:
That's interesting, because https://frontend.preview.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/classify is trying to load them from the static files directory in PFE. Is that because it isn't prefixing the URLs with the correct hostname?